src/pages/blog/multi-term-jquery-table-filter.md

```diff
@@ -44,32 +45,33 @@ The bulk of the code is all in the keyup function of the search textbox:
     })
     // Show the rows that match.
     .show();
-});`
-
-
+});
+```
 
 The split function is what allows the textbox to support multiple terms. In the fiddle, test it out on the right by typing in ‘cat two’. After typing in just ‘cat’ you should only see two records, but once you add another term, it shows the row that contains ‘two’ as well. Basically, it’s performing an OR based search on the terms being provided.
 
 Once the terms are split into the data array, the function gets the set of rows in the table. Then, it makes sure all rows are displaying and exits the function if the input box is empty. If it’s not empty, the filtering takes place by first hiding all of the rows, and then running a filter function. The filter function removes records from an array if the associated function returns false, and keeps them if it returns true. In this case, it will return true if the row passes jQuery “:contains” selector for any element of the data array of terms. Any one term is sufficient, since it’s an OR condition. If no term is matched, the function returns false and that row is not included. The “.show();” at the end is chained onto the result of the filter, so all of the rows that are found are displayed.
 
 Normally the “:contains” selector is case-sensitive, so searching for ‘cat’ would not find rows that only contained ‘Cat’. There’s not an easy way to fix this from within the function itself. Rather, you can [make jQuery :contains case-insensitive](https://css-tricks.com/snippets/jquery/make-jquery-contains-case-insensitive/), like this:
 
-`// make contains case insensitive globally // (if you prefer, create a new Contains or containsCI function)
+```js
+// make contains case insensitive globally // (if you prefer, create a new Contains or containsCI function)
 $.expr[":"].contains = $.expr.createPseudo(function(arg) {
     return function( elem ) {
         return $(elem).text().toUpperCase().indexOf(arg.toUpperCase()) >= 0;
     };
-});`
-
-
+});
+```
 
 Try it:
```

Comment:
I'm not sure how that used to work but it can be removed now. Just get rid of Try it and everything below it.